docs/concepts/abstractions/pod.md

```diff
@@ -0,0 +1,74 @@
+
+#### How Pods Manage Containers
+
+Pods are designed to support multiple cooperating processes (as application containers) that form a cohesive unit of service. The containers in a Pod are automatically co-located and co-scheduled on the same phyiscal or virtual machine in the cluster. The containers can share resources and dependencies, communicate with one another, and coordinate when and how they are terminated.
+
+Pods provide two kinds of shared resources for their constituent containers: *networking* and *storage*.
+
+##### Networking
+
+Each Pod is assigned a unique IP address. Every the container in a pod shares the network namespace, including the IP address and network ports. Containers *inside a Pod* can communicate with one another using `localhost`. When containers in a Pod communicate with entities *outside the Pod*, they must coordinate how they use the shared network resources (such as ports).
+
+##### Storage
+
+A Pod can specify a set of shared storage *volumes*. All containers in the pod can access the shared volumes, allowing those containers to share data. Volumes also allow persistent data in a pod to survive in case one of the containers within needs to be restarted. See [Volumes]() for more information on how Kubernetes implements shared storage in a Pod.
+
+### Working with Pods
+
+When a Pod gets created (directly or indirectly), it is scheduled to run on a [node]() in your cluster, and remains on that node until terminated or deleted. Should a node in the cluster fail, the Pods scheduled on that node are deleted after a timeout period. See [Termination](#pod-termination) for more details on how Pods terminate in Kubernetes.
+
+You'll rarely create or interact directly with individual Pods in Kubernetes--even singleton Pods. This is because Pods are designed as relatively ephemeral entities (as opposed to a durable one). A Pod won't survive a scheduling failure, a node failure, or an eviction due to a lack of resources or node maintenance. Thus, while it is possible to use Pod directly, it's far more common in Kubernetes to manage your pods using a higher-level abstraction called a *Controller*. 
+
+#### <a name="pods-and-controllers"></a> Pods and Controllers
+
+A Controller can create and manage multiple Pods for you, handling replication and rollout and providing self-healing capabilities at cluster scope (for example, if a node fails, a Controller might schedule an identical replacement Pod on a different node). 
+
+Some examples of Controllers that contain one or more pods include:
+
+* [Deployment]()
+* [StatefulSet]()
+* [DaemonSet]()
+
+In general, Controllers use a [Pod Template]() that you provide to create the Pods for which it is responsible.
+
+#### <a name="pod-termination"></a> Pod Termination
+
```

Comment:
Probably don't need to go into detail, but it wasn't clear what the system did that it waits to complete in the graceful termination.  Presumably it did something that it is waiting for, but it isn't clear what from the doc